core-services/image-mirroring/admin_redhat-developer_list.yaml

```diff
@@ -0,0 +1,38 @@
+kind: List
+apiVersion: v1
+items:
+- apiVersion: v1
+  kind: Namespace
+  metadata:
+    name: redhat-developer
+- apiVersion: authorization.openshift.io/v1
+  kind: RoleBinding
+  metadata:
+    name: image-puller
+    namespace: redhat-developer
+  roleRef:
+    name: system:image-puller
+  subjects:
+  - kind: SystemGroup
+    name: system:authenticated
```

Comment:
system:unauthenticated, are you sure you want that?

Reply:
per documentation, it seems this is what is wanted

Reply:
actually, there is both:
 `system:unauthenticated` and `system:authenticated` , this is taken from other project that has the same requirement.
The documentation is saying that is must have unauthenticated (annoymous access) allowed to the imagestream.
I guess that authenticated will not harm in this case and is preferable as it will allow normal user to pull or to `oc image info` without having to logout